src/components/SwiperFlatList/SwiperFlatList.js

```diff
@@ -215,6 +218,9 @@ const SwiperFlatList = React.forwardRef(
       paginationDefaultColor,
       paginationStyle,
       paginationStyleItem,
+      renderRightButton,
```

Comment:
This should follow the name conventions `paginationRenderLeft` and `paginationRenderRight`